backend/app/processing/vanna_integrator.py

```
@@ -0,0 +1,267 @@
+from pathlib import Path
+import sqlite3
+from typing import Dict, Any
+import os
+import shutil
```

Ruff (0.8.2)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)